README.md

```diff
@@ -8,56 +8,108 @@
 ![Status](https://nexus.lab.fiware.org/static/badges/statuses/iot-ultralight.svg)
 
-## Index
+An Internet of Things Agent for the Ultralight 2.0 protocol (with
+[AMQP](https://www.amqp.org/), [HTTP](https://www.w3.org/Protocols/) and
+[MQTT](https://mqtt.org/) transports). This IoT Agent is designed to be a bridge
+between Ultralight and the
+[NGSI](https://swagger.lab.fiware.org/?url=https://raw.githubusercontent.com/Fiware/specifications/master/OpenAPI/ngsiv2/ngsiv2-openapi.json)
+interface of a context broker.
+
+It is based on the
+[IoT Agent Node.js Library](https://github.com/telefonicaid/iotagent-node-lib).
+Further general information about the FIWARE IoT Agents framework, its
+architecture and the common interaction model can be found in the library's
+GitHub repository.
+
+This project is part of [FIWARE](https://www.fiware.org/). For more information
+check the FIWARE Catalogue entry for the
+[IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents).
+
+## Contents
+
+-   [Background](#background)
+-   [Install](#install)
+-   [Usage](#usage)
+-   [API](#api)
+-   [Testing](#testing)
+-   [Quality Assurance](#quality-assurance)
+-   [License](#license)
+
+## Background
+
+This _Internet of Things Agent_ is a bridge that can be used to communicate
+devices using the Ultralight 2.0 protocol and NGSI Context Brokers (like
+[Orion](https://github.com/telefonicaid/fiware-orion)). Ultralight 2.0 is a
+lightweight text based protocol aimed to constrained devices and communications
+where the bandwidth and device memory may be limited resources. This IoT Agent
+will provide different transport protocol bindings for the same protocol: HTTP,
+MQTT...
+
+As is the case in any IoT Agent, this one follows the interaction model defined
+in the
+[Node.js IoT Agent Library](https://github.com/telefonicaid/iotagent-node-lib),
+that is used for the implementation of the APIs found on the IoT Agent's North
+Port. Information about the architecture of the IoT Agent can be found on that
+global repository. This documentation will only address those features and
+characteristics that are particular to the Ultralight 2.0 IoT Agent.
+
+Additional information about operating the component can be found in the
+[Operations: logs and alarms](docs/operations.md) document.
+
+## Install
+
+Information about how to install the IoT Agent for Ultralight can be found at
+the corresponding section of the
+[Installation & Administration Guide](https://fiware-iotagent-ul.readthedocs.io/en/latest/installationguide).
+
+## Usage
+
+Information about how to use the IoT Agent can be found in the
+[User & Programmers Manual](https://fiware-iotagent-ul.readthedocs.io/en/latest/usermanual).
+
+## API
+
+Apiary reference for the Configuration API can be found
+[here](http://docs.telefonicaiotiotagents.apiary.io/#reference/configuration-api).
```

Comment:
Is this link still valid?

(This is a self-question :). I'll investigate it and answer)

Reply:
The Apiary document could be obsolete. The reference in the github itself: https://github.com/telefonicaid/iotagent-node-lib#configuration-api is the fresher and the one that should be cited.

CC: @AlvaroVega